testsuite/features/reposync/srv_sync_channels.feature

```diff
@@ -48,32 +48,19 @@ Feature: Be able to list available channels and enable them
     And I should see "Total time:" in the output
     And I should see "Repo URL:" in the output
 
-@susemanager
-  Scenario: Enable RHEL 8 channels for Rocky 8
-    When I execute mgr-sync "add channel rhel8-pool-x86_64"
-    And I execute mgr-sync "add channel res-8-updates-x86_64"
-    And I execute mgr-sync "add channel res-as-8-updates-x86_64"
-    And I execute mgr-sync "add channel res-cb-8-updates-x86_64"
-    And I execute mgr-sync "add channel res8-manager-tools-pool-x86_64"
-    And I execute mgr-sync "add channel res8-manager-tools-updates-x86_64"
+  Scenario: Enable a real base channel
+    When I execute mgr-sync "add channel sles12-sp4-pool-x86_64"
```

Comment:
Is there a reason to add those SLES12 SP4 channels or were they simply moved and GitHub does not display this correctly?

Reply:
Good catch, I don't think this is needed as we don't have it on other branches anymore, and it would sync SLES12 SP4 which we don't especially need
I'll remove that.